repository/src/main/java/org/apache/atlas/repository/store/graph/v2/MeshEntityNotifierImpl.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.atlas.repository.store.graph.v2;
+
+import org.apache.atlas.RequestContext;
+import org.apache.atlas.exception.AtlasBaseException;
+import org.apache.atlas.listener.EntityChangeListenerV2;
+import org.apache.atlas.model.instance.AtlasEntity;
+import org.apache.atlas.repository.graphdb.AtlasVertex;
+import org.apache.atlas.utils.AtlasPerfMetrics;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.*;
+
```

Comment:
can we reuse BusinessPolicyNotifierImpl?
I suggest renaming BusinessPolicyNotifierImpl to MinimalNotifierImpl

Reply:
Arpit had asked me to keep it seperate but even I think we can have one generic implementation. Will discuss with Arpit once again